manifests/params.pp

```diff
@@ -80,7 +80,7 @@
   # Based on the small number of inputs above, we can construct sane defaults
   # for pretty much everything else.
 
-  # Settings common to everything
+  # # Settings common to everything
```

Comment:
Is the extra comment mark a mistake here?

Reply:
Yes.  Was when i was commenting out staging to get everything to fail and then replace it

Reply:
Done